facade/nodeFacade.go

```diff
@@ -388,6 +395,27 @@ func (nf *nodeFacade) GetProof(rootHash string, address string) ([][]byte, error
 	return trie.GetProof(addressBytes)
 }
 
+// GetProofCurrentRootHash returns the Merkle proof for the given address and current root hash
+func (nf *nodeFacade) GetProofCurrentRootHash(address string) ([][]byte, []byte, error) {
+	rootHash := nf.blockchain.GetCurrentBlockHeader().GetRootHash()
```

Comment:
This can easily panic in the first moments when the blockchain struct contains a nil block header. Please add a check if nil test here

Reply:
Done.